src/main/java/org/apache/maven/plugins/changes/FeedGenerator.java

```diff
@@ -235,4 +260,35 @@ private static Date getDate(final String dateRelease, final DateFormat dateForma
             return new Date();
         }
     }
+
+    /**
+     * Simple {@link OutputStream} implementation that writes to a {@link Writer}.
+     *
+     * @since 1.3
```

Comment:
Not only is the version number hallucinated. It's irrelevant, useless cruft on a private inner class.